Sources/Models/MessageData.swift

```diff
@@ -26,7 +26,7 @@ import Foundation
 import class CoreLocation.CLLocation
 
 /// An enum representing the kind of message and its underlying data.
-public enum MessageData {
+public enum MessageData: Equatable {
```

Comment:
I don't think we should make `MessageData` conform to `Equatable` or `RawRepresentable`. Making changes to the example app should never require modifications in the core library